crates/biome_cli/src/logging.rs

```diff
@@ -33,7 +33,7 @@ pub fn setup_cli_subscriber(
         .transpose()
         .expect("Failed to create log file")
         .optional()
-        .or_else(std::io::stdout);
```

Comment:
This is intentional. 
- stdout -> for machines
- stderr -> for humans

Since logs are structured and predictable, they are for machines, hence stdout 

Can you please revert? Even if we're wrong, it isn't part of the PR

Reply:
I have better things to do than make and land another half dozen PRs so I guess you can submit separately if you care for it. But be aware your default log format is pretty, which is not machine readable, and any kind of stdout logging is incompatible with `--stdin-file-path`. It feels pretty silly to add `--log-file /dev/stderr`, something I have never had to type in my life for any other program.